Habitica Intents/Habitica Intents.entitlements

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.chabitica.TasksSiri</string>
```

Comment:
Is there a reason why you called it `chabitica` and `chabitrpg`? Also, wouldn't a more generic name be better for this? Since we probably need something similar for a today widget, which we'd like to do in the future.

Reply:
Yes, it must look strange. I was required to change names/groups because of swapping to my apple account for testing. It's the same in a few other places I had to convert. I'll replace the instances before my final commit. 

Your second comment makes sense. How's just group.habitica?